ui/components/component-library/modal-body/README.mdx

```diff
@@ -0,0 +1,80 @@
+import { Controls, Canvas } from '@storybook/blocks';
+
+import * as ModalBodyStories from './modal-body.stories';
+
+# ModalBody
+
+`ModalBody` is a container component that handles the scrolling of the `Modal` content.
+
+<Canvas of={ModalBodyStories.DefaultStory} />
+
+## Props
+
+The `ModalBody` accepts all props below as well as all [Box](/docs/components-ui-box--default-story#props) component props
+
+<Controls of={ModalBodyStories.DefaultStory} />
+
+### Children
+
+Use the `children` prop to pass in any valid React children. The `ModalBody`'s purpose is to handle the scrolling of the `Modal` content when the content extends beyond the viewport.
+
+<Canvas of={ModalBodyStories.Children} />
```

Comment:
Displaying stories using `id`s has been deprecated in favor of `Controls` https://storybook.js.org/docs/writing-docs/mdx